tests/koa-authenticator.spec.js

```diff
@@ -62,7 +63,7 @@ describe('Koa Escher Request Authenticator Middleware', function() {
 
 
   it('should throw HTTP 401 in case of problem during request capture', function*() {
-    let error = new Error('Request capture error');
+    let error = new AuthenticationError('Request capture error');
```

Comment:
It doesn't seem to be relevant. This shouldn't be an AuthenticationError because the next test will cover this issue. It should be another kind of error not an Auth one and it shouldn't handle with error code 401.